include/hw_isolation_event/openpower_hw_status.hpp

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "hw_isolation_event/event.hpp"
+
+namespace hw_isolation
+{
+namespace event
+{
+namespace openpower_hw_status
+{
+
+/**
+ * - Enums to indicate non-error reason for a hardwares deconfiguration.
```

Comment:
Need some documentaion to indiacte , the below data is based what file version of hostboot , to help for futute=re updates.

Reply:
OK make sense, I can add commit id which is used for the last update on the `src/include/usr/hwas/common/deconfigGard.H`